functorch/test/test_pythonkey.py

```diff
@@ -373,6 +373,10 @@ def f(x):
             def get_grads(args):
                 return pytree.tree_map(lambda x: x.grad, args)
 
+            # NB: We cache on function id, which is unreliable
+            # Can fix by using weakrefs, but not sure if it matters
+            clear_compile_cache()
```

Comment:
Why not add this to our global setUp in common_utils?
That way all the tests will have that.

Reply:
Hrmm... it's kind of a hack.